packages/iris-grid/src/sidebar/InputEditor.tsx

```diff
@@ -175,9 +178,13 @@ export default class InputEditor extends Component<
           data-testid="custom-column-formula"
         />
         {isEditorEmpty && !value && (
-          <div className="editor-placeholder text-muted">Column Formula</div>
+          <div className="editor-placeholder text-muted">
+            {placeholder ?? 'Column Formula'}
```

Comment:
I don't think we should show anything for placeholder if specified, then `CustomColumnInput` should be passing in "Column Formula", instead of having it in this component. Within the context of `InputEditor`, knowledge of what is being input doesn't make sense.

Reply:
Fixed